plugins/source/mixpanel/docs/tables/mixpanel_funnels.md

```diff
@@ -0,0 +1,20 @@
+# Table: mixpanel_funnels
+
+The primary key for this table is **_cq_id**.
+
+## Relations
+
+The following tables depend on mixpanel_funnels:
+  - [mixpanel_funnel_reports](mixpanel_funnel_reports.md)
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id (PK)|UUID|
+|_cq_parent_id|UUID|
+|project_id|Int|
+|funnel_id|Int|
+|name|String|
```

Comment:
Bit strange maybe, but these three keys together should be okay as a PK, right?

Reply:
`funnel_id` is supposed to be the PK but code has a bug so it's not applied. Will fix.